mmcls/models/heads/multi_task_head.py

```diff
@@ -59,22 +70,14 @@ def loss(self, feats: Tuple[torch.Tensor],
             dict[str, Tensor]: a dictionary of loss components
```

Comment:
Please update the loss documentation

> The features extracted from the backbone.

Ok

> Multiple stage inputs are acceptable but only the last stage
> will be used to classify. The shape of every item should be
> ``(num_samples, num_classes)``.

Not true anymore


```suggestion
            dict[str, Tensor]: a dictionary of loss components, each task loss key will be prefixed by the task_name like "task1_loss"
```